doc/user_guide/install.rst

```diff
@@ -43,28 +43,47 @@ context (right-click) menu  entries to start the `Jupyter Notebook <http://jupyt
 Quick instructions to install HyperSpy using Anaconda (Linux, MacOs, Windows)
 -----------------------------------------------------------------------------
 
+Anaconda is recommended for the best performance (it is compiled using Intel
+MKL libraries) and the easiest installation. The academic license is free.
+
+
 #. Download and install
-   `Anaconda. <https://store.continuum.io/cshop/anaconda/>`_
-   For MacOS, be sure to choose the graphical installer, not the command-line version. Anaconda is recommended for the best performance (it is compiled
-   using Intel MKL libraries) and the easiest intallation (all the required
-   libraries are included). The academic license is free. Then install Hyperspy via the 'pip' command (in the Mac Terminal,  usually to be found in "Applications>Utilities" or in a Unix terminal):
+   `Anaconda <https://store.continuum.io/cshop/anaconda/>`_. If you are not
+   familiar with Anaconda please refer to their
+   `User Guide <https://docs.continuum.io/anaconda/index#user-guide>`_ for
+   details.
+
+#. Then install HyperSpy executing the following `conda` command in the Anaconda
+   Prompt, Linux/Mac Terminal or Microsoft Windows Command Prompt. (This
+   depends on your OS and how you have installed Anaconda, see the `Anaconda
+   User Guide <https://docs.continuum.io/anaconda/index#user-guide>`_) for
+   details.
 
    .. code-block:: bash
 
-       $ conda config --add channels conda-forge
-       $ conda install traitsui
-       $ pip install hyperspy
+       $ conda install hyperspy
```

Comment:
I agree, I've put back the add channels line.

2016-07-17 20:28 GMT+01:00 Tomas Ostasevicius <notifications@github.com>:

> In doc/user_guide/install.rst
> <https://github.com/hyperspy/hyperspy/pull/1158#discussion_r71087824>:
>
> >
> >     .. code-block:: bash
> >
> > -       $ conda config --add channels conda-forge
> > -       $ conda install traitsui
> > -       $ pip install hyperspy
> > +       $ conda install hyperspy
>
> I doubt that - most likely the config of conda was read from a file, where
> it was added as one of the channels.
>
> I would still prefer the explicit version - this would be foul-proof for
> any (both old and new) conda setup, working for existing users too.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hyperspy/hyperspy/pull/1158/files/d58643883bd3ee85e724686212188abb4925a04a#r71087824>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA8aFww62rPln-weKBmNIP__d39Pk62Xks5qWoJHgaJpZM4JNSTj>
> .
>